src/wallet/test/wallet_tests.cpp

```diff
@@ -292,8 +292,8 @@ BOOST_FIXTURE_TEST_CASE(get_available_watch_only_credit, TestChain100Setup)
   {
     LOCK(cs_main);
     const CWalletTx *wallet_tx = m_wallet->GetWalletTx(watch_only_coinbase->GetHash());
-    // The stake is watch-only
-    BOOST_CHECK_EQUAL(wallet_tx->GetAvailableCredit(false, ISMINE_WATCH_ONLY), 10000 * UNIT);
+    // The stake is not watch-only
```

Comment:
The stake now keeps the script of the coin all the time, before it was replaced with the script passed.